cirq-google/cirq_google/devices/serializable_device_test.py

```diff
@@ -77,6 +77,41 @@ def test_repr_pretty(cycle, func):
     printer.text.assert_called_once_with(func(device))
 
 
+def test_metadata_correct():
+    qubits = cirq.GridQubit.rect(2, 3, left=1, top=1)
+    device_proto = cgdk.create_device_proto_for_qubits(
+        qubits=qubits,
+        pairs=[
```

Comment:
Might be clearer to define `pairs` outside this, then use it for the `device.metadata.qubit_pairs` assertion.

Reply:
Done.